phoenix-core/src/test/java/org/apache/phoenix/compile/QueryCompilerTest.java

```diff
@@ -1253,7 +1253,9 @@ public void testUnknownColumnInPKConstraint() throws Exception {
         }
     }
     
-    
+
+    // see PHOENIX-3534, now tables can have duplicate columns and they are removed implicitly
```

Comment:
This should still be detected with an exception thrown.

Reply:
This test passes, I have remove the ignore annotation.